content/news/2018/04/navcoin-core-manifestos/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "NavCoin Core Manifestos, Cold Staking Testing."
+date: 2018-04-11T21:03:55+12:00
+draft: false
+author: "NavCore"
+description: ""
+resources: []
+categories: ["Nav News"]
+slug: ""
+type: "news"
+feature_image: "Wed-11th.png"
+show_title_text: false
+---
+This week the core team have started creating NavCoin Core manifesto documents, and we see a fantastic example of how members without coding skills can contribute to the project.
+<!--more-->
+### NavCoin Core Manifesto’s
+The Core Content team are currently working closely with developers to create a suite of NavCoin Core Manifesto’s. The purpose of these Manifesto’s is to make clear:
+
+
+* What the core contributor’s roles are
```

Comment:
isn't it contributors instead of contributor's? It's not possesive in this case is it?

Reply:
I think it mights be `contributors'` because its plural and possessive of the role ?

This `NavCoin Core Manifesto’s` should become `NavCoin Core Manifestos`.